### ISSUE TITLE
Thread Performance Checker

### ISSUE THREAD

**Issue by rastaman111** [marmelroy/PhoneNumberKit#646]
Hello! 
After I open a new screen, I parse the array of phones and during the parse, a purple warning occurs

Thread Performance Checker: Thread running at QOS_CLASS_USER_INTERACTIVE waiting on a lower QoS thread running at QOS_CLASS_DEFAULT. Investigate ways to avoid priority inversions PID: 7946, TID: 99584

**Comment by bguidolim** [comment]
Could to provide the steps to reproduce or a small reproducible piece of code?

**Comment by rastaman111** [comment]
I think this is no longer relevant. The solution was to complete the Podfile and update to the latest version of the pod
